src/components/Navigation/Navigation.jsx

```diff
@@ -118,11 +123,27 @@ export default class Navigation extends React.Component {
         inputSelector: '.navigation__search-input'
       });
 
-      window.addEventListener('keyup', e => {
-        if (e.which === 9 && e.target.classList.contains('navigation__search-input')) {
-          this._openSearch();
+      // Keydown so we can "stop" the event if need be. 
+      window.addEventListener("keydown", e => {
+        // Short circuit, as we only care about the "naked" key
+        if (e.shiftKey || e.ctrlKey || e.metaKey) return;
+      
+        switch (e.which) {
+          case 9: // `tab` key
+            if (e.target.classList.contains("navigation__search-input"))
+              this._openSearch();
+            break;
+          case 191: // `/` key
+            !this._searchOpen && this._openSearch();
+            e.preventDefault(); // to block entering a `/` if the input is already in focus.
```

Comment:
just wondering, if there is another input on the page. what will happen if they focus on it and type `api/cli`. It would mean the input receives `api` and then rest of it will go to search input: `cli` :D 
@montogeek do you think we have such input anywhere on the docs?

Reply:
Not that I know